pkg/graph/resolvers/model_plan.resolvers.go

```diff
@@ -113,6 +114,11 @@ func (r *modelPlanResolver) OperationalNeeds(ctx context.Context, obj *models.Mo
 	return OperationalNeedCollectionGetByModelPlanIDLOADER(ctx, obj.ID)
 }
 
+// LastModifiedDts is the resolver for the lastModifiedDts field.
+func (r *modelPlanResolver) LastModifiedDts(ctx context.Context, obj *models.ModelPlan) (*time.Time, error) {
```

Comment:
Can you please make some tests for this? Since the logic lives in SQL, it would be good to have some assertions that it returns the dates as expected.